### DISCUSSION TITLE
Difference between AR and Modelviewer

### DISCUSSION THREAD

**Discussion by divclass1** [google/model-viewer#4818] [Q&A]
Hi,

I tried to show a sofa in AR, but there is a difference between the MV editor and AR. In the editor the model looks good, but in AR it's bugging. Is this an issue with the 3D model or modelviewer?  I use the latest version (3.5.0)

Thanks in advance!
![Modelviewer](https://github.com/google/model-viewer/assets/120512726/1314b8ce-c4b8-4910-aa7f-f44ba45fe343)
![ARview](https://github.com/google/model-viewer/assets/120512726/89a2874c-32e2-42c1-832b-6343a48957e1)

**Comment by elalish** [comment]
For AR we *always* need to know which of the three AR modes you're using (or better yet, how each one compares). If this is iOS, then you're talking about Quick Look - I'd guess you've set your model for double-sided because your triangles are inverted and I think USDZ doesn't have double-sided. I'd strongly recommend avoiding double-sided materials, as they only hide modeling errors like this and even where it renders properly, it's half as fast.
